target/linux/ath79/image/generic.mk

```diff
@@ -979,6 +979,22 @@ define Device/sitecom_wlr-7100
 endef
 TARGET_DEVICES += sitecom_wlr-7100
 
+define Device/sitecom_wlr-8100
+  ATH_SOC := qca9558
```

Comment:
Please rename this to "SOC" due to 95030ac0b673165772daf27128359eaaf536d457.

Reply:
Fixed!